src/components/radio/Radio.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import styles from './Radio.css';
+
+const Input = (props) => <input type="radio" { ...props } />;
+
+const Radio = ({ label, className, ...rest }) => {
```

Comment:
Can we use same strategy as in the TextInput where all props will be spreaded to the root and the radio specific props are applied to the radio element?

Reply:
After doing this, please adjust the README.

Reply:
Done